stock_prices/stock_prices.py

```diff
@@ -3,7 +3,21 @@
 import argparse
 
 def find_max_profit(prices):
-  pass
+    buy = prices[0]
+    sell = 0
+    profit = -1
+    for n in range(0, len(prices)-1):
```

Comment:
I think it would help to start the for loop at the second stock price instead of the first